DXMainClient/DXGUI/Multiplayer/GameListBox.cs

```diff
@@ -332,6 +353,18 @@ public override void Draw(GameTime gameTime)
                         height, txPasswordedGame.Width, txPasswordedGame.Height),
                         Color.White);
                 }
+                else
+                {
+                    Texture2D txGameDifficulty = txGameDifficultyIcons[hostedGame.GameDifficulty];
+
+                    if (txGameDifficulty != null && hostedGame.GameDifficulty > 0) // Don't show GameDifficulty Any
```

Comment:
No, why? You're making an assumption about what difficulties will be set by modder. You already have no image (or you should not have an image), why add an additional check?.

```suggestion
                    if (txGameDifficulty != null)
```